scg-docker/Dockerfile

```diff
@@ -25,23 +25,24 @@ ARG LIB=${FUSEKI_DIR}/lib
 ARG AGENTS=${FUSEKI_DIR}/agents
 ARG SBOMS=/opt/telicent/sbom
 
-COPY entrypoint.sh .
+COPY scg-docker/entrypoint.sh .
+COPY scg-docker/logback.xml .
+
 RUN \
     mkdir -p $LOGS $DATA $CONFIGS $LIB $AGENTS $SBOMS && \
     chown -R fuseki ${FUSEKI_DIR} && \
     chmod a+x entrypoint.sh
 
 # Build location of the jars we need.
 ## Collected jars
-ARG JARS=target/dependency
+ARG JARS=scg-docker/target/dependency
 COPY ${JARS} lib
-ARG AGENTS=target/agents
+ARG AGENTS=scg-docker/target/agents
 COPY ${AGENTS} agents
 ARG PROJECT_VERSION
-ARG SBOM=target/docker-${PROJECT_VERSION}-bom.json
+ARG SBOM=scg-docker/target/docker-${PROJECT_VERSION}-bom.json
 COPY ${SBOM} /opt/telicent/sbom/smart-cache-graph-${PROJECT_VERSION}-bom.json
 
-ADD logback.xml .
```

Comment:
Why is this removed?

Reply:
Not removed. Moved up to line 28 alongside the addition of entrypoint.sh